### ISSUE TITLE
Directly stream file contents in Message.php

### ISSUE THREAD

**Issue by suushiemaniac** [botman/botman#443]
- BotMan Version: 1.5.7
- PHP Version: 7.0.20
- Messaging Service(s): Slack RTM
- Cache Driver: N/A (?)

### Description:
I would like to have a possibility to directly inject file contents (i.e. a raw string) into the `Message` object.
In my scenario I use BotMan to create a RTM Slack bot that processes user requests (via API.AI) and relays them to a technical API. The content returned from the API is then displayed to the user.

### Steps To Reproduce:
Currently, for a given API output `$output`, my code looks like this:
```php
$tmpFile = '/path/to/file';
file_put_contents($tmpFile, $output);

$bot->reply(Message::create('Meaningful message content')->filePath($tmpFile));
unlink($tmpFile);
```

I would much prefer the following: (take the method name `fileRaw` as a suggestion, you can of course consider anything else)
```php
$bot->reply(Message::create('Meaningful message content')->fileRaw($output));
```

P.S.: Thanks for this awesome library :blush:

**Comment by mpociot** [comment]
@suushiemaniac I like that approach. Do you want to help me with a pull request for this new feature?

**Comment by suushiemaniac** [comment]
Yeah sure! I only recently started working with your library, so I'll get to it as soon as possible
